### DISCUSSION TITLE
Devices drop off the network? Allow devices to join and put the device in the pairing mode to reconnect

### DISCUSSION THREAD

**Discussion by hnykda** [Koenkk/zigbee2mqtt#11950] [Show and tell]
Hey.

I don't know why, but from time to time, _some_ devices just drop off my zigbee network. I have slae.sh coordinator and about 50 various devices such as blinds, IKEA FLOALT, bulbs, Tuya TRVs, xiaomi/aqara temp sensors... The devices work as far as I can tell (e.g. their LED works, or can be controlled via bind switches such as IKEA Tradfi controller and IKEA FYRTUR blinds), but zigbee2mqtt do not receive the message (e.g. when I check the mqtt topic, I don't see any log/action). I can't find why that happens (it's not some specific device, it's any of them)...

I found out that you don't have to somehow disconnect/remove the device and reset everything. What works is to just put the zigbee2mqtt coordinator into pairing mode ("Allow devices to join") and then put the device that is off the network into the pairing mode again. It seems zigbee2mqtt then pick it up again. If the device was part of a group, I had to remove the device (e.g. the blind) from the group and then add the device to the group again (I didn't have to bind the remote to the group, which make sense, as I didn't do any change there).

I of course hate that this is happening in the first place, but at least quick recovery exists :shrug: .

**Comment by bednim** [comment]
Hi, drop outs are really annoying and to be fair - never had them with conbee II coordinator - but had other issues with it - so replaced it with slae.sh stick - and at the very beginning the performance was really great (better than conbee II), after some time - just over a month - I am getting terrible zigbee performance - devices drop out (lots of them - even mains powered), terrible lag, unresponsiveness. zigbee became very unreliable for me which is a pity. 

i am looking to switch onto different stick just to make sure this isn't just slae.sh stick issue.

**Reply by hnykda** [reply]
I doubt it will be solved by a different stick :shrug: :-D

**Reply by bednim** [reply]
can you please try and explain how have you positioned your coordinator? how long is your extension cable? any tricks perhaps?

I have already changed my wifi/zigbee channels, got tons of routers on the ZB network - yet - experience is terrible.

**Reply by hnykda** [reply]
Nothing special, a very cheap extension hub that I had lying around. I would be super surprised if that was the cause, as I tried some other ones and it was the same.

I actually suspect (and some others too) that more routers is only getting more congestion and complications, so I don't have any repeaters if I don't really have to have them (and I don't, my house is quite compact).
